### PR TITLE
Add the bmi160-spi-demo

### PR DESCRIPTION
**Pull request by DanielHo-BS**
1. Minor update quick start.
2. Add the bmi160-spi-demo
        1. nvidia/bmi160_spi_demo/images
        2. nvidia/bmi160_spi_demo/index.rst
        3. nvidia/bmi160_spi_demo/spi_test.rst
        4. nvidia/bmi160_spi_demo//spi_demo.rst
        5. nvidia/index.rst >> update menu

Please check it, thx.

### CODE REVIEW

File: nvidia/bmi160_spi_demo/index.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.. _bmi160_spi_demo:
+
+BMI160 SPI Demo
+###############
+
+| The chapter shows **How to Test SPI** and **How to Make SPI Demo** with **BMI160**.
```
Comment (evshary) [inline nvidia/bmi160_spi_demo/index.rst]: You can mention the supported model of BMI160 here.
Supported model: ROScube Pico series

File: nvidia/index.rst [hunk 2]
```diff
@@ -22,3 +22,4 @@ The supported models:
    gmsl_camera/index.rst
    extension_box/index.rst
    advanced/index.rst
+   bmi160_spi_demo/index.rst
```
Comment (evshary) [inline nvidia/index.rst]: You can put BMI160 document under "The peripherals"

**Review by evshary — CHANGES_REQUESTED** [review]

**Review by DanielHo-BS — COMMENTED** [review]
Finish the Request changes.

Please check it, thx.

File: nvidia/bmi160_spi_demo/index.rst (hunk 1 above)
Comment (DanielHo-BS) [reply nvidia/bmi160_spi_demo/index.rst]: Done.

File: nvidia/index.rst (hunk 2 above)
Comment (DanielHo-BS) [reply nvidia/index.rst]: OK

**Comment by evshary** [comment]
LGTM.

**Pull request merged**